src/Resources/views/CRUD/list__action_delete.html.twig

```diff
@@ -10,12 +10,20 @@ file that was distributed with this source code.
 #}
 
 {% if admin.hasAccess('delete', object) and admin.hasRoute('delete') %}
+    {% set _button_mode = sonata_config.getOption('button_mode') %}
     <a
         href="{{ admin.generateObjectUrl('delete', object, actions.link_parameters|default([])) }}"
         class="btn btn-sm btn-default delete_link"
         title="{{ 'action_delete'|trans({}, 'SonataAdminBundle') }}"
     >
-        <i class="fa fa-times" aria-hidden="true"></i>
-        {{ 'action_delete'|trans({}, 'SonataAdminBundle') }}
+        {% if _button_mode == 'both' or _button_mode == 'icon' %}
+            <i class="fa fa-times" aria-hidden="true"></i>
+            <span class="sr-only">
+                {{ 'action_delete'|trans({}, 'SonataAdminBundle') }}
+            </span>
+        {% endif %}
+        {% if _button_mode == 'both' or _button_mode == 'text' %}
+            {{ 'action_delete'|trans({}, 'SonataAdminBundle') }}
+        {% endif %}
```

Comment:
Here, with `both` the text is display two times, one times inside `sr-only` and one time without.

One way to fix this could be
```suggestion
        {% if _button_mode != 'text' %}
            <i class="fa fa-times" aria-hidden="true"></i>
        {% endif %}
        {% if _button_mode == 'icon' %}
            <span class="sr-only">
                {{ 'action_delete'|trans({}, 'SonataAdminBundle') }}
            </span>
        {% else %}
            {{ 'action_delete'|trans({}, 'SonataAdminBundle') }}
        {% endif %}
```